### PR TITLE
[YUNIKORN-2616] Remove unused bool return from PreemptionPredicates()

### PR DESCRIPTION
**Pull request by ryankert01**
### What is this PR for?
The predicate manager method PreemptionPredicates() returns two values an int and boolean. The boolean is false if the integer is -1 and true for 0 or llarger. There is no need for the boolean as the -1 already indicates the same

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [x] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2616

### How should this be tested?
Github CI

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: pkg/plugin/predicates/predicate_manager_test.go [hunk 1]
```diff
@@ -69,8 +69,8 @@ func TestPreemptionPredicatesEmpty(t *testing.T) {
 	node := framework.NewNodeInfo()
 	node.SetNode(&v1.Node{})
 	victims := make([]*v1.Pod, 0)
-	index, ok := predicateManager.PreemptionPredicates(pod, node, victims, 0)
-	assert.Check(t, !ok, "check should have failed")
+	index := predicateManager.PreemptionPredicates(pod, node, victims, 0)
+	assert.Check(t, index == -1, "check should have failed")
 	assert.Equal(t, index, -1, "wrong index")
```
Comment (chenyulin0719) [inline pkg/plugin/predicates/predicate_manager_test.go:74]: Only 1 assert required. 

-> assert.Equal(t, index, -1, "should not find a victim after preemption check.")

File: pkg/plugin/predicates/predicate_manager_test.go [hunk 2]
```diff
@@ -69,8 +69,8 @@ func TestPreemptionPredicatesEmpty(t *testing.T) {
...
 }
```
Comment (chenyulin0719) [inline pkg/plugin/predicates/predicate_manager_test.go:75]: Only 1 assert required.

-> assert.Equal(t, index, -1, "should not find any victim index after preemption check")

File: pkg/plugin/predicates/predicate_manager_test.go [hunk 3]
```diff
@@ -115,17 +115,17 @@ func TestPreemptionPredicates(t *testing.T) {
 	node.AddPod(victims[3])
 
 	// all but 1 existing pod should need removing
-	index, ok := predicateManager.PreemptionPredicates(pod, node, victims, 1)
-	assert.Check(t, ok, "check failed")
+	index := predicateManager.PreemptionPredicates(pod, node, victims, 1)
+	assert.Check(t, index != -1, "check failed")
 	assert.Equal(t, index, 2, "wrong index")
```
Comment (chenyulin0719) [inline pkg/plugin/predicates/predicate_manager_test.go:120]: Ditto

-> assert.Equal(t, index, 2, "wrong index")

File: pkg/plugin/predicates/predicate_manager_test.go [hunk 4]
```diff
@@ -115,17 +115,17 @@ func TestPreemptionPredicates(t *testing.T) {
...
 	// try again, but with too many resources requested
 	pod = newResourcePod(framework.Resource{MilliCPU: 1500, Memory: 15000000})
 	pod.Name = "largepod"
 	pod.UID = "largepod"
 
-	index, ok = predicateManager.PreemptionPredicates(pod, node, victims, 1)
-	assert.Check(t, !ok, "check should have failed")
+	index = predicateManager.PreemptionPredicates(pod, node, victims, 1)
+	assert.Check(t, index == -1, "check should have failed")
 	assert.Equal(t, index, -1, "wrong index")
```
Comment (chenyulin0719) [inline pkg/plugin/predicates/predicate_manager_test.go:129]: Ditto.

-> assert.Equal(t, index, -1, "should not find any victim index after preemption check")

**Review by chenyulin0719 — CHANGES_REQUESTED** [review]
Some minor changes.

File: pkg/plugin/predicates/predicate_manager_test.go (hunk 2 above)
Comment (ryankert01) [reply pkg/plugin/predicates/predicate_manager_test.go:74]: You're right! Done. Btw, thanks for your review!

**Review by pbacsko — APPROVED** [review]
+1

**Review by chia7712 — APPROVED** [review]

**Pull request closed without merging**